packages/react-composites/src/composites/CallWithChatComposite/CallWithChatControlBar.tsx

```diff
@@ -11,7 +11,7 @@ import { callControlsContainerStyles } from '../CallComposite/styles/CallPage.st
 import { useCallWithChatCompositeStrings } from './hooks/useCallWithChatCompositeStrings';
 import { ChatAdapter } from '../ChatComposite';
 import { ChatButtonWithUnreadMessagesBadge } from './ChatButtonWithUnreadMessagesBadge';
-import { BaseCustomStyles, ControlBarButtonStyles } from '@internal/react-components';
+import { BaseCustomStyles, ControlBarButtonStyles, HoldButton } from '@internal/react-components';
```

Comment:
Don't we need to conditionally compile hold button import and usage here?

Reply:
yup thanks for the catch here! 😊